CoffeeChat/Controller/TalkingPointsViewController.swift

```diff
@@ -14,23 +14,7 @@ class TalkingPointsViewController: UIViewController {
     private weak var delegate: OnboardingPageDelegate?
     // TODO: change when networking with backend
     private var displayedTalkingPoints: [SimpleOnboardingCell] = []
-    private let interests: [Interest] = [
-         Interest(name: "Art", categories: ["painting", "crafts", "embroidery"], imageName: "art"),
-         Interest(name: "Business", categories: ["entrepreneurship", "finance", "VC"], imageName: "business"),
-         Interest(name: "Dance", categories: ["urban, hip hop", "ballet", "swing"], imageName: "dance"),
-         Interest(name: "Design", categories: ["UI/UX", "graphic", "print"], imageName: "design"),
-         Interest(name: "Fashion", categories: nil, imageName: "fashion"),
-         Interest(name: "Fitness", categories: ["working out", "outdoors", "basketball"], imageName: "fitness"),
-         Interest(name: "Food", categories: ["cooking", "eating", "baking"], imageName: "food"),
-         Interest(name: "Humanities", categories: ["history", "politics"], imageName: "humanities"),
-         Interest(name: "Music", categories: ["instruments", "producing", "acapella"], imageName: "music"),
-         Interest(name: "Photography", categories: ["digital", "analog"], imageName: "photography"),
-         Interest(name: "Reading", categories: nil, imageName: "reading"),
-         Interest(name: "Sustainability", categories: nil, imageName: "sustainability"),
-         Interest(name: "Tech", categories: ["programming", "web/app development"], imageName: "tech"),
-         Interest(name: "Travel", categories: ["road", "trips", "backpacking"], imageName: "travel"),
-         Interest(name: "TV & Film", categories: nil, imageName: "tvfilm")
-    ]
+    private let interests: [Interest] = Constants.Options.interests
```

Comment:
I don't think you need the type annotation. If this variable isn't used too often, I would directly use `Constants.Options.interests` since you wouldn't clutter the namespace by creating a new class variable.